haystack/constants.py

```diff
@@ -10,6 +10,7 @@
 ID = getattr(settings, 'HAYSTACK_ID_FIELD', 'id')
 DJANGO_CT = getattr(settings, 'HAYSTACK_DJANGO_CT_FIELD', 'django_ct')
 DJANGO_ID = getattr(settings, 'HAYSTACK_DJANGO_ID_FIELD', 'django_id')
+HAYSTACK_DOCUMENT_FIELD = getattr(settings, 'HAYSTACK_DOCUMENT_FIELD', 'text')
```

Comment:
It seems like this could be `DOCUMENT_FIELD` following the other settings losing the `HAYSTACK_` prefix when retrieved.